lib/opencog-chatbot.conf

```diff
@@ -22,19 +22,19 @@ PROMPT                = ""
 
 # These file paths assume that you have done `make install` after
 # compiling; these modules are found in `/usr/local/lib/opencog`.
-MODULES               = opencog/modules/libbuiltinreqs.so,
-                        opencog/modules/libscheme-shell.so,
-                        opencog/modules/libpy-shell.so,
-                        opencog/modules/libattention-types.so,
-                        opencog/modules/libembodiment-types.so,
-                        opencog/modules/libnlp-types.so,
-                        opencog/modules/libspacetime-types.so,
-                        opencog/modules/libLGDictModule.so,
-                        opencog/modules/libPersistModule.so,
-                        opencog/modules/libPythonModule.so,
-                        opencog/modules/libQueryModule.so,
-                        opencog/modules/libRuleEngineModule.so,
-                        opencog/modules/libSuRealModule.so
+MODULES               = opencog/cogserver/modules/libbuiltinreqs.so,
+                        opencog/cogserver/modules/libscheme-shell.so,
+                        opencog/cogserver/modules/libpy-shell.so,
+                        opencog/cogserver/modules/libattention-types.so,
+                        opencog/cogserver/modules/libembodiment-types.so,
+                        opencog/cogserver/modules/libnlp-types.so,
+                        opencog/cogserver/modules/libspacetime-types.so,
+                        opencog/cogserver/modules/libLGDictModule.so,
+                        opencog/cogserver/modules/libPersistModule.so,
+                        opencog/cogserver/modules/libPythonModule.so,
+                        opencog/cogserver/modules/libQueryModule.so,
+                        opencog/cogserver/modules/libRuleEngineModule.so,
+                        opencog/cogserver/modules/libSuRealModule.so
```

Comment:
i haven't tried installing it and running the cogserver, so yes their might be some issue. 

I will make a pr for https://github.com/opencog/opencog/issues/1968, I am testing now